src/main/java/hudson/tasks/Mailer.java

```diff
@@ -687,22 +688,16 @@ public FormValidation doSendTestMail(
                 @QueryParameter boolean useSsl, @QueryParameter boolean useTls, @QueryParameter String smtpPort, @QueryParameter String charset,
                 @QueryParameter String sendTestMailTo) throws IOException {
             try {
-                // TODO 1.590+ Jenkins.getActiveInstance
-                final Jenkins jenkins = Jenkins.getInstance();
-                if (jenkins == null) {
-                    throw new IOException("Jenkins instance is not ready");
-                }
+                Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
Shouldn't this be MANAGE? The button to do this action is displayed to MANAGE according the changes in the jelly file.

Reply:
Your correct, not sure how that got missed!